conda/recipes/ucx-py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = 'ucx-py' %}
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version = environ.get('CONDA_PY', '38') %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.2').split('.')[:2]) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../../..
```

Comment:
Should we use `git_url` here instead?

Reply:
Either will work, but yeah we can switch. For context this came from [here]( https://github.com/rapidsai/cusignal/blob/600eb63d48ecb21ef4a374bbb91a0f6fa34b4828/conda/recipes/cusignal/meta.yaml#L12 )

Reply:
Updated